pkg/util/quotapool/example_test.go

```diff
@@ -55,24 +55,24 @@ func ExampleIntPool_AcquireFunc() {
 		ctx context.Context, qp *IntPool,
 	) (j *job, alloc *IntAlloc, err error) {
 		alloc, err = qp.AcquireFunc(ctx, func(
-			ctx context.Context, v int64,
-		) (fulfilled bool, took int64) {
+			ctx context.Context, pi PoolInfo,
+		) (took uint64, err error) {
 			sortJobs()
 			// There are no more jobs, take 0 and return.
 			if jobs[0] == nil {
-				return true, 0
+				return 0, nil
 			}
 			// Find the largest jobs which can be run.
 			for i := range jobs {
 				if jobs[i] == nil {
 					break
 				}
```

Comment:
I take it you'll add some logic to ensure that some job exists which is less than pi.Capacity in a later commit?